actix-cache-derive/tests/test_cacheable_derive.rs

```diff
@@ -127,3 +128,10 @@ fn test_default_ttl_stale_ttl_version_work() {
     assert_eq!(message.cache_stale_ttl(), 55);
     assert_eq!(message.cache_version(), 0);
 }
+
+#[derive(Cacheable, Serialize)]
+struct MessageV {
```

Comment:
Why did you add this struct?

Reply:
Who knows :smile: 
I guess I wanted to add some tests with it.